Makefile

```diff
@@ -148,10 +144,6 @@ checksum:
 
 # ------------------------------------------------------------------------------
 
-.PHONY: docs
```

Comment:
Is `make docs` no longer valid?

Reply:
As far as I understand, yes. See above discussion: https://github.com/helm/helm/pull/5458#discussion_r269648921

Reply:
@adamreese @bacongobbler Lets discuss at the dev meeting today. I am confused npw whether the target is still needed or not.

Reply:
Following a discussion on slack with @adamreese @flynnduism, the docs targets are still required to generate the help docs for the web documentation.